source/_integrations/ssdp.markdown

```diff
@@ -3,6 +3,7 @@ title: Simple Service Discovery Protocol (SSDP)
 description: Discover integrations on the network using the SSDP protocol.
 ha_category:
   - Network
+ha_iot_class: Local Polling
```

Comment:
SSDP is based on mDNS and listens for announcements on a UDP broadcast.

```suggestion
ha_iot_class: Local Push
```

Reply:
Thanks for explanation, I learning about inner workings of a lot of integrations

Reply:
It is an world full of different tech 😄